src/panels/lovelace/hui-view.js

```diff
@@ -133,6 +135,16 @@ class HUIView extends PolymerElement {
     const elements = config.cards.map((cardConfig) => {
       const element = createCardElement(cardConfig);
       element.hass = this.hass;
+      if (this.editMode) {
```

Comment:
to keep this function simpler, the moment you know you're not in edit mode, you can return the element as you know you no longer need processing. It's called a guard clause.

```
if (!this.editMode) { return element; }
```